bin/run_webapp.sh

```diff
@@ -26,11 +29,6 @@ if [ "$1" == "--dev" ]; then
     cd /app/webapp-django/ && ${CMDPREFIX} python manage.py runserver 0.0.0.0:8000
 
 else
-    echo "GUNICORN_WORKERS=${GUNICORN_WORKERS}"
-    echo "GUNICORN_WORKER_CLASS=${GUNICORN_WORKER_CLASS}"
-    echo "GUNICORN_MAX_REQUESTS=${GUNICORN_MAX_REQUESTS}"
-    echo "GUNICORN_MAX_REQUESTS_JITTER=${GUNICORN_MAX_REQUESTS_JITTER}"
-    echo "PORT=${PORT}"
```

Comment:
I added `set -x` so it spits out the variables as they're set and we don't need to additionally echo them. That reduces a lot of error-prone code.